Dockerfile.slim

```diff
@@ -20,6 +20,10 @@ RUN wget https://raw.githubusercontent.com/kubernetes/helm/master/scripts/get &&
   ./get && \
   rm get
 
+RUN wget -O /tmp/get-pip.py https://bootstrap.pypa.io/get-pip.py && \
```

Comment:
This looks to install python 2.7 which is EOL in 10 months---could we install something more future-proof?